asset/scss/_footer.scss

```diff
@@ -0,0 +1,39 @@
+.wrap-footer {
+    background-image: url( $network );
+
+    &.footer-slim {
```

Comment:
I realize this is copied from `base.scss`, meaning I probably didn't notice this in an earlier review, but is the ampersand needed here? It seems like it would compile the same without it, but my SASS is pretty rusty.

Reply:
The ampersand causes that selector to be compiled as a second class with the containing rule - so by the time this gets into `base.css` the selector rule is `.wrap-footer.footer-slim` (one element with both class names, which is what's needed to override the style library we're importing).

By comparison the rule below this, without the ampersand, ends up compiling to `.wrap-footer .wrap-logo-lib` (with a space, so one element inside another)